.github/workflows/checks.yaml

```diff
@@ -2,13 +2,19 @@ name: Tests
 
 on:
     pull_request:
-        branches: [main]
+        branches:
+            - main
     push:
-        branches: [main]
+        branches:
+            - main
 jobs:
-    docker_tests:
+    tests:
```

Comment:
Curious why we're renaming the job

Reply:
I forgot why I did, but theres no docker so maybe why I did it.